guides/v2.1/contributor-guide/templates/basic_template.md

```diff
@@ -14,12 +14,14 @@ You can start off by editing the local version of this file using markdown langu
 
 ## Metadata parameters
 
-Add the following {% glossarytooltip 3f0f2ef1-ad38-41c6-bd1e-390daaa71d76 %}metadata{% endglossarytooltip %} parameters at the top of your page. We use this data when generating the DevDocs for the following:
+Add the following metadata parameters at the top of your page. We use this data when generating the DevDocs for the following:
```

Comment:
Add that we use YAML for the metadata in front matter.

Reply:
Added note to the opening text.